io/disk/bonnie.py

```diff
@@ -64,15 +64,16 @@ def setUp(self):
         process.run('./configure')
         build.make(self.srcdir)
 
-        self.part_obj = Partition(self.disk, mountpoint=self.scratch_dir)
-        self.log.info("Test will run on %s", self.scratch_dir)
-        self.log.info("Unmounting the disk/dir before creating file system")
-        self.part_obj.unmount()
-        self.log.info("creating file system")
-        self.part_obj.mkfs(self.fstype)
-        self.log.info("Mounting disk %s on directory %s",
-                      self.disk, self.scratch_dir)
-        self.part_obj.mount()
+        if self.disk is not None:
+            self.part_obj = Partition(self.disk, mountpoint=self.scratch_dir)
+            self.log.info("Test will run on %s", self.scratch_dir)
+            self.log.info("Unmounting disk/dir before creating file system")
+            self.part_obj.unmount()
+            self.log.info("creating file system")
```

Comment:
Can you please add the kind of file system you are creating?
Example: `"Creating file system %s", fstype`